src/lib/plot/series/line-series.js

```diff
@@ -36,15 +36,27 @@ const STROKE_STYLES = {
 
 const defaultProps = {
   strokeStyle: 'solid',
-  opacity: 1
+  opacity: 1,
+  curve: null
 };
 
 const propTypes = {
   ...AbstractSeries.propTypes,
-  strokeStyle: React.PropTypes.oneOf(Object.keys(STROKE_STYLES))
+  strokeStyle: React.PropTypes.oneOf(Object.keys(STROKE_STYLES)),
+  curve: React.PropTypes.string
 };
 
 class LineSeries extends AbstractSeries {
+
+  _renderLine(data, x, y, curve) {
+    let line = d3Shape.line();
+    if (curve !== null && d3Shape[curve]) {
+      line = line.curve(d3Shape[curve]);
```

Comment:
How about allowing users to import d3-shape and configure and submit their own curve function:
```javascript
if (curve !== null) {
  if (typeof curve === 'string' && d3Shape[curve]) {
    line = line.curve(d3Shape[curve]);
  } else if (typeof curve === 'function') {
    line = line.curve(curve);
  }
}
```